code/src/java/pcgen/core/ShieldProf.java

```diff
@@ -42,7 +42,7 @@
 	@Override
 	public int compareTo(final Object o1)
 	{
-		return getKeyName().compareToIgnoreCase(((ShieldProf) o1).getKeyName());
+		return getKeyName().compareToIgnoreCase(((VarScoped) o1).getKeyName());
```

Comment:
I'd rather this not be changed.  I know you like generics, but in general these Comparable<Object> items are hideous.  The usage needing these should be eliminated, and it would be nice to have this throw a cast exception if someone really does try to compare to a non-ShieldProf